From 450d3260f36e647b8952570c54a281e288e742f7 Mon Sep 17 00:00:00 2001
From: kalle <kalyan.alle@intel.com>
Date: Thu, 13 Jan 2022 15:30:31 +0800
Subject: [PATCH] libdrm chanes to support rpl-s platform

---
 intel/i915_pciids.h   | 11 +++++++++++
 intel/intel_chipset.c |  1 +
 2 files changed, 12 insertions(+)

diff --git a/intel/i915_pciids.h b/intel/i915_pciids.h
index c7cdbfc4..43c3302c 100644
--- a/intel/i915_pciids.h
+++ b/intel/i915_pciids.h
@@ -505,4 +505,15 @@
 	INTEL_VGA_DEVICE(0x4551, info), \
 	INTEL_VGA_DEVICE(0x4541, info)
 
+/* RPL-S */
+#define INTEL_RPLS_IDS(info) \
+	INTEL_VGA_DEVICE(0xA780, info), \
+	INTEL_VGA_DEVICE(0xA781, info), \
+	INTEL_VGA_DEVICE(0xA782, info), \
+	INTEL_VGA_DEVICE(0xA783, info), \
+	INTEL_VGA_DEVICE(0xA788, info), \
+	INTEL_VGA_DEVICE(0xA789, info), \
+	INTEL_VGA_DEVICE(0xA78A, info), \
+	INTEL_VGA_DEVICE(0xA78B, info)
+
 #endif /* _I915_PCIIDS_H */
diff --git a/intel/intel_chipset.c b/intel/intel_chipset.c
index 5aa4a2f2..0570ba2b 100644
--- a/intel/intel_chipset.c
+++ b/intel/intel_chipset.c
@@ -35,6 +35,7 @@ static const struct pci_device {
 	uint16_t gen;
 } pciids[] = {
 	/* Keep ids sorted by gen; latest gen first */
+	INTEL_RPLS_IDS(12),
 	INTEL_ICL_11_IDS(11),
 	INTEL_CNL_IDS(10),
 	INTEL_CFL_IDS(9),
-- 
2.17.1

