From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kalyan Alle <kalyan.alle@intel.com>
Date: Mon, 13 Sep 2021 16:50:56 +0800
Subject: [PATCH] mixins gfx changes to enable drm_hwc and drm_minigbm

for zero copy
---
diff --git a/groups/graphics/auto/BoardConfig.mk b/groups/graphics/auto/BoardConfig.mk
index 22d15a6..62e17bb 100755
--- a/groups/graphics/auto/BoardConfig.mk
+++ b/groups/graphics/auto/BoardConfig.mk
@@ -20,10 +20,10 @@ endif

 USE_OPENGL_RENDERER := true
 USE_INTEL_UFO_DRIVER := false
-BOARD_GPU_DRIVERS := i965 swrast virgl iris
+BOARD_GPU_DRIVERS := i965 swrast virgl iris kmsro
 BOARD_USE_CUSTOMIZED_MESA := true

-BOARD_GPU_DRIVERS ?= i965 swrast virgl iris
+BOARD_GPU_DRIVERS ?= i965 swrast virgl iris kmsro
 ifneq ($(strip $(BOARD_GPU_DRIVERS)),)
 TARGET_HARDWARE_3D := true
 TARGET_USES_HWC2 := true
-- 
2.17.1

