From 8006af240c33996012bfd013cc8f0eb01ba9ef02 Mon Sep 17 00:00:00 2001
From: Basanagouda Koppad <basanagoudax.n.koppad@intel.com>
Date: Fri, 17 Dec 2021 07:17:25 +0530
Subject: [PATCH] Add SOONG_NAMESPACE for sensor mediation flag

Tracked-On:
Signed-off-by: Basanagouda Koppad <basanagoudax.n.koppad@intel.com>

diff --git a/groups/sensors/mediation/BoardConfig.mk b/groups/sensors/mediation/BoardConfig.mk
index 33f8cad..81f1e7b 100644
--- a/groups/sensors/mediation/BoardConfig.mk
+++ b/groups/sensors/mediation/BoardConfig.mk
@@ -1,7 +1,10 @@
 USE_SENSOR_MEDIATION_HAL := true
 
+
 {{#disable_static_sensor_list}}
-MEDIATION_HAL_DISABLE_STATIC_SENSOR_LIST := true
+SOONG_CONFIG_NAMESPACES += senPlugin
+SOONG_CONFIG_senPlugin += MEDIATION_HAL_DISABLE_STATIC_SENSOR_LIST
+SOONG_CONFIG_senPlugin_MEDIATION_HAL_DISABLE_STATIC_SENSOR_LIST := true
 {{/disable_static_sensor_list}}
 
 BOARD_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/sensors/mediation
-- 
2.34.1

