From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Basil Chew <basil.chew@intel.com>
Date: Thu, 19 Aug 2021 16:56:28 +0800
Subject: [PATCH] Mixins update - mixins_sriov_changes

---
diff --git a/caas/extra_files/media/auto_hal.in b/caas/extra_files/media/auto_hal.in
index f307863..5c7be0a 100644
--- a/caas/extra_files/media/auto_hal.in
+++ b/caas/extra_files/media/auto_hal.in
@@ -14,7 +14,7 @@ case "$(cat /proc/fb)" in
                 ;;
         *)
                 echo "software codec"
-                setprop vendor.intel.video.codec software
+                setprop vendor.intel.video.codec hardware
                 ;;
 esac
 }
-- 
2.17.1

