From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Basil Chew <basil.chew@intel.com>
Date: Wed, 9 Feb 2022 15:10:13 +0800
Subject: [PATCH] Mixins update - Port SRIOV graphics and media installation
 for host

---
 caas/device.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/caas/device.mk b/caas/device.mk
index 7d19c10..6cef47d 100644
--- a/caas/device.mk
+++ b/caas/device.mk
@@ -292,6 +292,7 @@ PRODUCT_COPY_FILES += $(LOCAL_PATH)/thermald.service:$(PRODUCT_OUT)/scripts/ther
 PRODUCT_COPY_FILES += device/intel/civ/host/vm-manager/scripts/setup_host.sh:$(PRODUCT_OUT)/scripts/setup_host.sh
 PRODUCT_COPY_FILES += device/intel/civ/host/vm-manager/scripts/guest_time_keeping.sh:$(PRODUCT_OUT)/scripts/guest_time_keeping.sh
 PRODUCT_COPY_FILES += device/intel/civ/host/vm-manager/scripts/start_flash_usb.sh:$(PRODUCT_OUT)/scripts/start_flash_usb.sh
+PRODUCT_COPY_FILES += device/intel/civ/host/vm-manager/scripts/sriov_setup_host.sh:$(PRODUCT_OUT)/scripts/sriov_setup_host.sh
 PRODUCT_COPY_FILES += vendor/intel/fw/trusty-release-binaries/rpmb_dev:$(PRODUCT_OUT)/scripts/rpmb_dev
 PRODUCT_COPY_FILES += $(LOCAL_PATH)/wakeup.py:$(PRODUCT_OUT)/scripts/wakeup.py
 ##############################################################
-- 
2.17.1

